app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -225,7 +160,7 @@ class RoutePresenter(
         }
     }
 
-    private fun findTransitionId(@IdRes from: Int, @IdRes to: Int): Int? {
+    fun findTransitionId(@IdRes from: Int, @IdRes to: Int): Int? {
```

Comment:
This should be an `protected` `abstract` method in `RoutePresenter` and implemented in `AutofillRoutePresenter` and `AppRoutePresenter`.